### PR TITLE
Make debug output more orderly.

### PR DESCRIPTION
**Pull request by gfldex**
see: https://gfldex.wordpress.com/2021/06/23/being-pragmat-ish/

### CODE REVIEW

**Comment by gfldex** [comment]
I would like to add that this is the first PR do help with logging. Having a `lib/Discrod/Debug.rakumod` is a precondition for things to follow.

File: META6.json [hunk 1]
```diff
@@ -1,55 +1,63 @@
...
+    "API::Discord::Message::ButReal": "lib/API/Discord/Message.rakumod",
+    "API::Discord::Object": "lib/API/Discord/Object.rakumod",
+    "API::Discord::Permissions": "lib/API/Discord/Permissions.rakumod",
+    "API::Discord::RESTy": "lib/API/Discord/HTTPResource.rakumod",
+    "API::Discord::Types": "lib/API/Discord/Types.rakumod",
+    "API::Discord::User": "lib/API/Discord/User.rakumod",
+    "API::Discord::User::ButReal": "lib/API/Discord/User.rakumod",
+    "API::Discord::WebSocket": "lib/API/Discord/WebSocket.rakumod",
+    "API::Discord::WebSocket::Messages": "lib/API/Discord/WebSocket/Messages.rakumod",
+    "API::Discord::Debug": "lib/API/Discord/Debug.rakumod",
```
Comment (kawaii) [inline META6.json:45]: Just a minor thing that triggers my OCD a little, would suggest moving this line up to L#29, trying to keep this list ordered alphabetically. :)

**Comment by gfldex** [comment]
2nd part is now done. The user can divert output to $*ERR to a Supply and do some filtering via:

```
    use API::Discord::Debug;

    react whenever debug-say().merge(debug-print()) {
        when HEARTBEAT { $*ERR.print: $_ }
        when CONNECTION { note now.Datetime.Str, ' ', $_ }
        when /your term here/ { .&do-stuff() }
        default {}
    }
```

This allows to deal with error messages in the main react block. All symbols can be kept lexical to avoid collision with other modules. The raku-bot is using this now in the following fassion, which works nicely.

```
start react {
        whenever $discord.messages -> $message {
            next unless $channels ~~ $message.channel.name;

            $result.emit: [ $message.channel.&augment, $message.author.&augment, $message.content ];
        }
        # [...]
       {
                use API::Discord::Debug;

                whenever debug-say().merge(debug-print()) {
                        when HEARTBEAT { }
                        default { .note }
                }
        }
}
```

**Comment by kawaii** [comment]
Thanks for your contribution, I will try and get around to testing this either tomorrow or Monday and look to getting it merged during the week hopefully.

**Pull request merged**